core/src/queue.rs

```diff
@@ -199,8 +198,13 @@ impl Queue {
         let entry = match self.txs.entry(hash) {
             Entry::Occupied(mut old_tx) => {
                 // MST case
+                let signatures_amount_before = old_tx.get().signatures().len();
                 assert!(old_tx.get_mut().merge_signatures(tx));
-                info!("Signature added to existing multisignature transaction");
```

Comment:
Most of the time this would entry produce logs for just transaction duplicate, received through gossiping, so i decided to only log if signatures amount increased.